src/tds/util.c

```diff
@@ -354,6 +361,7 @@ tdserror (const TDSCONTEXT * tds_ctx, TDSSOCKET * tds, int msgno, int errnum)
 		msg.sql_state = tds_alloc_client_sqlstate(msgno);
 		
 		msg.oserr = errnum;
+		msg.osstr = errnum ? (char*) s_StrError(errnum) : NULL;
```

Comment:
This field is used only in ctlib, why not moving the conversion int -> string into ctlib?

Reply:
The field was in principle generically useful, but I concede that the other APIs aren't actually in a position to do anything with it; simplified accordingly.

Reply:
Merged